xdsl/dialects/llvm.py

```diff
@@ -91,6 +92,79 @@ def opaque():
     def typed(type: Attribute):
         return LLVMPointerType([type, NoneAttr()])
 
+    def is_typed(self):
+        return not isinstance(self.type, NoneAttr)
+
+
+@irdl_op_definition
+class GEPOp(Operation):
+    name = "llvm.getelementptr"
+
+    ptr: Annotated[Operand, LLVMPointerType]
+    ssa_indices: Annotated[VarOperand, IntegerType]
+
+    elem_type: OptOpAttr[Attribute]
+
+    rawConstantIndices: OpAttr[DenseArrayBase]
+
+    inbounds: OptOpAttr[UnitAttr]
+
+    result: OpResult
+
+    @staticmethod
+    def get(ptr: SSAValue | Operation,
+            indices: list[int] | None = None,
```

Comment:
I am guessing that you are assuming that these indices already follow the MLIR standard (min int where the ssavalue should be used). Please document this somewhere, either on this constructor, or the min int thing on the class and then say that this constructor assume indices to be in that form already.